modules/peers.js

```diff
@@ -407,22 +476,22 @@ Peers.prototype.list = function (options, cb) {
 
```

Comment:
We still perform consensus calculations here in `Peers.prototype.list` - why if we have new functions for that?